frontend/src/citizen-frontend/messages/MessagesPage.tsx

```diff
@@ -27,9 +25,7 @@ import ThreadView from './ThreadView'
 import { getReceivers, sendMessage } from './api'
 import { MessageContext } from './state'
 
-const FullHeightContainer = styled(Container)`
-  height: calc(100vh - ${headerHeightDesktop}px);
-`
```

Comment:
It's not a "full height container" anymore, so you should change the name. Or just use a plain `<div>` instead.